include/tunnler/room_member.h

```diff
@@ -86,13 +90,13 @@ class RoomMember final {
      * Sends a chat message to the room.
      * @param message The contents of the message.
      */
-    void SendChatMessage(const std::string message);
+    void SendChatMessage(const std::string& message);
 
 	/**
      * Sends a WiFi packet to the room.
      * @param packet The WiFi packet to send.
      */
-    void SendWifiPacket(const WifiPacket& packet);
+    void SendWifiPacket(WifiPacket& packet);
```

Comment:
Because there's a very very awful
````
stream.Write(reinterpret_cast<char*>(wifi_packet.data.data()), wifi_packet.data.size());
````

which would have to become either
````
stream.Write(const_cast<char*>(reinterpret_cast<const char*>(wifi_packet.data.data())), wifi_packet.data.size());
````
or
````
stream.Write((char*)wifi_packet.data.data(), wifi_packet.data.size());
````

Suggestions?